components/user_settings/advanced/user_settings_advanced.jsx

```diff
@@ -509,6 +551,93 @@ export default class AdvancedSettingsDisplay extends React.Component {
             }
         }
 
+        let deactivateAccountSection = '';
+        let makeConfirmationModal = '';
+        const currentUser = UserStore.getCurrentUser();
+        const config = getConfig(store.getState());
```

Comment:
We have `config` on [index.js](https://github.com/cpanato/mattermost-webapp/blob/eb7d7287136b62793843bc391feea7d4ad52034c/components/user_settings/advanced/index.js#L10).  I think we can get the `EnableUserDeactivation` from there and just pass as props.